spec/ttml1.xml

```diff
@@ -7107,6 +7107,9 @@ context of a layout or <loc href="#element-vocab-type-content">Content</loc> ele
 context, and not in the non-presentation, declaration context
 of a referentiable <el>style</el> element.</p>
 </note>
+<note role="clarification">
+<p>Step (4) is not applied for the style property <code>["http://www.w3.org/ns/ttml#styling", "lineHeight"]</code> when its value expression is <code>normal</code>, which is not considered relative.</p>
```

Comment:
This note has to be deleted. We can't (and shouldn't) list (or take note of) exceptional processing regarding what is or what is not relative in this context. If it is felt that a note is absolutely necessary, then it should say nothing more than "See individual style properties for information regarding on what values are considered to be relative for the purpose of interpreting step (4) above."

Reply:
@skynavga Happy to use _See individual style properties for information regarding what values are considered to be relative for the purpose of interpreting step (4) above._

Would that address all your comments?

Reply:
Just for this note, but not the other items. As I am preparing a PR against this PR, I will include this change.